tests/cephfs/cephfs_utils.py

```diff
@@ -69,40 +109,43 @@ def get_clients(self, build):
         for client in self.clients:
             node = client.node
             if node.pkg_type == "rpm":
-                if "7." in distro_ver:
-                    node.exec_command(
-                        sudo=True,
-                        cmd="curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py --insecure",
-                        long_running=True,
-                    )
-                    node.exec_command(
-                        sudo=True,
-                        cmd="python get-pip.py --trusted-host files.pythonhosted.org "
-                        "--trusted-host pypi.org --trusted-host pypi.python.org",
-                        long_running=True,
-                    )
-                else:
-                    node.exec_command(sudo=True, cmd="yum install -y python2-pip")
-                out, rc = node.exec_command(sudo=True, cmd="rpm -qa | grep -w attr")
+                out, rc = node.exec_command(
+                    sudo=True, cmd="rpm -qa | grep -w 'attr\\|xattr'"
+                )
                 output = out.read().decode()
                 output.split()
                 if "attr" not in output:
                     node.exec_command(sudo=True, cmd="yum install -y attr")
                 node.exec_command(
-                    sudo=True, cmd="yum install -y gcc python2-devel", check_ec=False
+                    sudo=True, cmd="yum install -y gcc python3-devel", check_ec=False
                 )
 
-                out, rc = node.exec_command(
-                    sudo=True, cmd="pip2  list --format=columns"
-                )
-                output = out.read().decode()
-                output.split()
-                if "crefi" not in output:
-                    node.exec_command(sudo=True, cmd="pip2 install crefi")
+                if "xattr" not in output:
+                    cmds = [
+                        "subscription-manager repos --enable rhel-7-server-optional-rpms "
+                        "--enable rhel-server-rhscl-7-rpms",
+                        "yum -y install @development",
+                        "yum -y install rh-python36",
+                        "yum -y install rh-python36-numpy rh-python36-scipy",
+                        "yum -y install rh-python36-python-tools rh-python36-python-six",
+                        "yum install -y libffi libffi-devel",
```

Comment:
using a list and iterating makes it easier here too

Reply:
Hi Rakesh,

I am iterating and executing the commands

Reply:
i meant, you could use `list` like in the previous file and in the below loop you could use something like this 
```
for pkg in pkgs:
     cmd = f"yum install -y {pkg}"
```     
or all at once 
```
cmd = "yum install -y " + ' '.join(pkgs)
```